lib/java_buildpack/container/tomcat/tomcat_geode_store.rb

```diff
@@ -44,8 +44,10 @@ def release
         credentials = @application.services.find_service(FILTER, KEY_LOCATORS, KEY_USERS)['credentials']
         user = credentials[KEY_USERS].find { |u| cluster_operator?(u) }
 
-        @droplet.java_opts.add_system_property 'gemfire.security-username', user['username']
-        @droplet.java_opts.add_system_property 'gemfire.security-password', user['password']
+        File.open(@droplet.root + 'gemfire.properties', 'w') do |f|
```

Comment:
How does the session manager find this file. Is it possible to move it to `@droplet.sandbox`?

If it needs to be in the `root` can we create it in the sandbox and symlink it to the root?

Reply:
The properties file is picked up in the current directory, home directory, or the CLASSPATH (https://geode.apache.org/docs/guide/12/configuring/running/default_file_specs.html).  We tried a few locations, but were only ever able to get it to pick up in the `@droplet.root`.  I'm not sure if we ever tried the `sandbox` directly though.  I know we tried it in `sandbox/conf`.  Let me double check what all those directories are and experiment some more.  Is the `@droplet.sandbox` the ideal location for the file?

Reply:
Deleted the addition of this file altogether in favor of pulling the credentials directly from `VCAP_SERVICES` in our ClientAuth class.